packages/tlschema/src/store-migrations.ts

```diff
@@ -63,5 +66,40 @@ export const storeMigrations = createMigrationSequence({
 				}
 			},
 		},
+		{
+			id: Versions.FixIndexKeys,
+			scope: 'record',
```

Comment:
can you add a noop down migration please. This will enable tlsync to support older clients and it won't force people to refresh immediately after deploying.